rainbowstream/rainbow.py

```diff
@@ -410,9 +411,33 @@ def tweet():
     """
     Tweet
     """
-    t = Twitter(auth=authen())
-    t.statuses.update(status=g['stuff'])
+    # Regex to check if tweet contains '--i' pattern
+    pattern = r'(.*) \-\-\i\ (.+)'
```

Comment:
The escape before "i" doesn't work in my Python 3.7 environment 🤔

Reply:
yeah, removed the escape before i